sides/thai-chile-brussels-sprouts.md

```diff
@@ -0,0 +1,42 @@
+# Thai chile Brussels sprouts
```

Comment:
if this is a side, I would separate the rice cooking portion of it otherwise it is a meal

Reply:
Then a meal it is.